cognitive/src/main/scala/com/microsoft/azure/synapse/ml/geospatial/AzureMapsHelpers.scala

```diff
@@ -64,6 +124,72 @@ trait HasAddressInput extends HasServiceParams with HasSubscriptionKey with HasU
   }
 }
 
+trait BatchReverseAddressGeocoding extends HasServiceParams
+  with HasSubscriptionKey with HasURL with HasLatLonPairInput{
+  protected def prepareEntity: Row => Option[AbstractHttpEntity]
+
+  protected def contentType: Row => String = { _ => "application/json" }
+
+  protected def inputFunc(schema: StructType): Row => Option[HttpRequestBase] = {
```

Comment:
I see you override this function as opposed to prepareEntity and prepareEntity doesn't have an impl. Is this your desired behavior? Seems like you might be able to remove code by just overriding prepareEntity instead

Reply:
Will ping on IM to better understand this

Reply:
As discussed. Removed `PrepareEntity` and `ContentType` for now. Will circle back to refactor `CognitiveServicesInput` to support both auth mechanisms